Content.Server/Zombies/ZombifyOnDeathSystem.cs

```diff
@@ -221,6 +234,16 @@ public void ZombifyEntity(EntityUid target, MobStateComponent? mobState = null)
                 _sharedHands.RemoveHand(target, hand.Name);
             }
             RemComp<HandsComponent>(target);
+            // No longer waiting to become a zombie:
+            RemComp<PendingZombieComponent>(target);
+
+            // Zombiefication takes a second. Groan on the floor for a bit.
+            if (EntityManager.TryGetComponent<StatusEffectsComponent>(target, out var status))
+            {
+                _stunSystem.TryStun(target, TimeSpan.FromSeconds(1.0), true, status);
+                _stunSystem.TryKnockdown(target, TimeSpan.FromSeconds(0.8), true,
+                    status);
+            }
```

Comment:
```suggestion
```

This should be removed or changed.

 As it is right now you die > revive as a zombie > get up for 1/4 of a second > get stunned for 1 second > get up. It feels very awkward, especially if a zombie is clawing you on death and you turn into a zombie on the next hit, which registers as someone falling down twice in a row. 

They're already dead before turning into a zombie so there's no need for a stun.

Alternatively, it should be a number of seconds _after_ death before they turn if you wanted to time gate the transformation.